numpy/_core/multiarray.py

```diff
@@ -894,7 +898,7 @@ def vdot(a, b):
     >>> 1*4 + 4*1 + 5*2 + 6*2
     30
 
-    """
+    """  # noqa: E501
```

Comment:
Only needed for the URL. Perhaps Sphinx would allow a line break here, but I wasn't sure.